src/test/java/nextstep/subway/AcceptanceTest/SectionAcceptanceTest.java

```diff
@@ -0,0 +1,200 @@
+    private final StationApi stationApi = new StationApi();
+
+    @LocalServerPort
+    int port;
+
+    @Autowired
+    DatabaseCleaner databaseCleaner;
+
+    Long 광교역, 강남역;
+    Long 신분당선;
+
+    @BeforeEach
+    public void setUp() {
+        if (RestAssured.port == RestAssured.UNDEFINED_PORT) {
+            RestAssured.port = port;
+        }
+
+        광교역 = stationApi.createId("광교역");
+        강남역 = stationApi.createId("강남역");
+        신분당선 = lineApi.create("신분당선", "bg-red-600", 강남역, 광교역).jsonPath().getLong("id");
+    }
+
+    @AfterEach
+    void cleanUp() {
+        databaseCleaner.execute();
+    }
+
+    /**
+     * When 지하철 노선에 새로운 지하철역 등록을 요청한다.
+     * Then 지하철 노선에 지하철역이 등록된다.
+     */
+    @DisplayName("노선에 구간을 등록한다.")
+    @Test
+    void create() {
+        // when
+        Long 양재역 = stationApi.createId("양재역");
+        ExtractableResponse<Response> response = lineApi.addSection(신분당선, 강남역, 양재역, 5);
+
+        // then
+        assertAll(
+                () -> assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value()),
+                () -> assertThat(lineApi.findById(신분당선).jsonPath().getList("stations.name")).contains("양재역")
+        );
+    }
+
+    /**
+     * Given 하행역을 상행 종점역과 동일하게 입력한다.
+     * When 지하철 노선에 지하철역 등록을 요청한다.
+     * Then 지하철 노선에 지하철역이 등록된다.
+     * Then 지하철 노선의 상행 종점역이 새로운 역으로 변경된다.
+     */
+    @DisplayName("새로운 역을 상행 종점으로 등록한다.")
+    @Test
+    void createNewEndUpStation() {
+        // given
+        Long 신논현역 = stationApi.createId("신논현역");
+
+        // when
+        ExtractableResponse<Response> response = lineApi.addSection(신분당선, 신논현역, 강남역, 5);
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        assertThat(getLineStationNames(신분당선).get(0)).isEqualTo("신논현역");
+    }
+
+    /**
+     * Given 상행역을 하행 종점역과 동일하게 입력한다.
+     * When 지하철 노선에 지하철역 등록을 요청한다.
+     * Then 지하철 노선에 지하철역이 등록된다.
+     * Then 지하철 노선의 하행 종점역이 새로운 역으로 변경된다.
+     */
+    @DisplayName("새로운 역을 하행 종점으로 등록한다.")
+    @Test
+    void createNewEndDownStation() {
+        // given
+        Long 호매실역 = stationApi.createId("호매실역");
+
+        // when
+        ExtractableResponse<Response> response = lineApi.addSection(신분당선, 광교역, 호매실역, 5);
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        List<String> lineStationNames = getLineStationNames(신분당선);
+        assertThat(lineStationNames.get(lineStationNames.size() - 1)).isEqualTo("호매실역");
+    }
+
+    /**
+     * Given 두개의 구간이 등록되어 있는 지하철 노선이 있다.
+     * When 지하철 노선에 새로운 구간 등록을 요청한다.
+     * Then 지하철 노선에 지하철역이 등록된다.
+     */
+    @DisplayName("노선에 구간을 여러개 등록한다.")
+    @Test
+    void createSections() {
+        // Given
+        lineApi.addSection(신분당선, 강남역, stationApi.createId("양재역"), 50);
+
+        // when
+        Long 신논현역 = stationApi.createId("신논현역");
+        ExtractableResponse<Response> response = lineApi.addSection(신분당선, 신논현역, 강남역, 10);
```

Comment:
하행역 기준으로 역을 추가하면 실패하는 것 같아요..! 
제가 잘못 테스트 했을수도 있기에 한번 확인해주시면 감사하겠습니다 :)
`강남 - 양재 - 신논현(new) - 광교`
```suggestion
    void createSections() {
        // Given
        lineApi.addSection(신분당선, 강남역, stationApi.createId("양재역"), 50);

        // when
        Long 신논현역 = stationApi.createId("신논현역");
        ExtractableResponse<Response> response = lineApi.addSection(신분당선, 신논현역, 광교역, 1);
```

Reply:
추가로 구간이 잘 등록되는지 확인하기 위해 Sections 도메인에 대한 도메인 테스트도 추가해주시면 감사하겠습니다 🙇

Reply:
도메인테스트를 추가하였는데 원하는 방향인지 모르겠네요 도움이 되었으면 좋겠습니다~~!!

Reply:
https://github.com/next-step/atdd-subway-admin/pull/818#issuecomment-1159621673

요 코멘트 관련해서 도메인 테스트를 추가해보면 어떨까했던 리뷰였습니다 🙏